pkgs/development/tools/misc/gf/default.nix

```diff
@@ -0,0 +1,66 @@
+{ lib, stdenv, makeWrapper
+
+  ,fetchgit, libX11, pkg-config, gdb
+
+  ,freetypeSupport ? false, freetype
+  ,extensionFolder ? null
+}:
+
+let
+  revision = "e0d6d2f59344f853a4a204d5313db6b6a5e5de7d";
+in
+
+stdenv.mkDerivation rec {
+  pname = "gf";
+  version = "2+${builtins.substring 0 7 revision}";
+
+  src = fetchgit {
+    url = "https://github.com/nakst/gf.git";
+    rev = "${revision}";
```

Comment:
Put the opaque revision directly here.

Better: use `fetchFromGitHub` instead of a crude `fetchgit`.

Reply:
done